src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionEncryptOption.cs

```diff
@@ -54,6 +56,21 @@ internal static SqlConnectionEncryptOption Parse(string value)
             }
         }
 
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionEncryptOption.xml' path='docs/members[@name="SqlConnectionEncryptOption"]/TryParse/*' />
+        public static bool TryParse(string value, out SqlConnectionEncryptOption result)
+        {
+            result = null;
+            try
+            {
+                result = Parse(value);
+                return true;
+            }
+            catch
```

Comment:
Exceptions are expensive. I would move the Parse logic into TryParse, refactoring it a bit to not throw an exception. Parse can then call TryParse and throw an exception if it fails.